### DISCUSSION TITLE
V23正式版登录/锁屏界面屏幕键盘无法输入标点符号

### DISCUSSION THREAD

**Discussion by canhong12** [linuxdeepin/developer-center#10730] [Troubleshooting | 故障排查]
**SUMMARY | 问题概要**
deepin V23正式版登录/锁屏界面上，无法用屏幕键盘输入标点符号。

**SOFTWARE/OS VERSIONS | 软件/系统版本信息**
Distribution: deepin V23 Release
Package:
dde-session-shell 6.0.21

**PRE-REQUIREMENT | 预置条件**
开机进入用户登录界面，或在日常使用中进入锁屏界面。

**STEPS TO REPRODUCE | 复现步骤**
打开屏幕键盘，点击屏幕键盘上面的标点符号按钮，切换到数字键盘亦可。

**OBSERVED RESULT | 观察到的结果**
屏幕键盘上面的标点符号按钮颜色变深，没有变回浅色，且密码输入框没有显示键入新的字符。而点击字母、数字按钮则没有这个问题。
![登录屏幕键盘问题](https://github.com/user-attachments/assets/14f9dd99-2c4e-4af6-9d56-51585b5847ff)

**EXPECTED RESULT | 期望的结果**
屏幕键盘上面的标点符号按钮在点击时变深，松开鼠标后变成浅色，密码输入框有新字符回显。

**ADDITIONAL INFORMATION | 额外补充**

**Comment by BLumia** [comment]
cc @babyfengfjx
